src/components/Textbox/index.js

```diff
@@ -1,20 +1,116 @@
 import React from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import Textarea from 'react-textarea-autosize';
-import styles from './Textbox.module.scss';
+import PlainTextarea from 'react-textarea-autosize';
 
 import { Text } from 'components';
 
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`;
+
+const TextareaContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-grow: 1;
+
+  /* added this to avoid bug where the top and right :active border disappeared */
+  /* padding: ${props => props.theme.baseSpacing}; */
+  /* margin: ${props => -props.theme.baseSpacing}; */
+`;
+
+const TextareaInfo = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Overlay = styled.div`
+  position: absolute;
+  top: ${props => props.theme.baseSpacing};
+  right: ${props => props.theme.baseSpacing};
+  margin-top: ${props => props.theme.baseSpacing};
+  margin-right: ${props => props.theme.baseSpacing};
+  border: ${props => props.theme.baseBorder};
+  border-radius: ${props => props.theme.baseBorderRadius};
+  background: ${props => props.theme.brandWhite};
+  padding: ${props => props.theme.sSpacing};
+`;
+
+const Textarea = styled(PlainTextarea)`
+  resize: ${props => (props.resizable === 'true' ? 'vertical' : 'none')};
+  flex-grow: 1;
+  width: 100%;
+
+  ${({ minHeight }) => minHeight && `min-height: ${minHeight};`} ${({
+    maxHeight
+  }) => maxHeight && `max-height: ${maxHeight};`}
+
+  overflow: auto;
+  background-color: ${props => props.theme.baseInputBackground};
+  border: ${props => props.theme.baseBorder};
+  border-radius: ${props => props.theme.baseBorderRadius};
+  font-size: ${props => props.theme.bodyFontSize};
+  font-family: inherit;
+  padding: ${props => props.theme.baseSpacing};
+
+  ${props =>
+    props.error &&
+    `
+    border-color: transparent;
+    box-shadow: 0 0 0 2px ${props.theme.brandDestructive};
+    outline: none
+  `} ${props =>
+    props.disabled &&
+    `
+      background-color: ${props.theme.brandNearWhite};
+      border: none;
+
+      &::placeholder {
+        color: ${props.theme.brandLightGrey};
+      }
+  `}
+
+
+  &::placeholder {
+    color: ${props => props.theme.brandGrey};
+  }
+
+  &:focus {
+    border-color: transparent;
+    background: ${props => props.theme.brandWhite};
+    box-shadow: 0 0 0 2px ${props => props.theme.brandBlue};
```

Comment:
Do we have a variable for the box-shadow applied to inputs on focus?

Reply:
It doesn't appear that we don't, so I made some 😄